cegui/src/ImageCodecModules/STB/stb_image.cpp

```diff
@@ -3298,7 +3298,10 @@ static stbi_uc *tga_load(stbi *s, int *x, int *y, int *comp, int req_comp)
       skip(s, tga_palette_start );
       //   load the palette
       tga_palette = (unsigned char*)malloc( tga_palette_len * tga_palette_bits / 8 );
-      if (!tga_palette) return epuc("outofmem", "Out of memory");
+      if (!tga_palette) {
```

Comment:
opening brackets on new line please

Reply:
Fixed

Reply:
If old implementation of STB is buggy to have a mem leak like this, should we consider to update the whole `stb_image.cpp` file with a newer one? Latest version maybe.

Reply:
@manhnt9  yes but that shouldnt be part of this change should it?